API/api.py

```diff
@@ -1233,6 +1233,109 @@ def get_event_groups(year: int, event: str, token: str = Depends(check_token_act
     return [Group(**group).dict(exclude={"join_code", "settings", "events", "owner_group_id", "admin_group_id", "member_group_id", "group_id"}) for group in eventGroups]
 
 
+@app.get("/group/{group_name}/{event_code}/ScoutingReport", tags=["groups"])
```

Comment:
```suggestion
@app.get("/Group/{group_name}/{event_code}/ScoutingReport", tags=["groups"])
```
Change the path for the Group to be capitalized